index.js

```diff
@@ -0,0 +1,44 @@
+const express = require('express');
+require("dotenv").config();
+const productRouter=require("./routes/product-router.js");
+const categoryRouter=require("./routes/category-router.js");
+const cartRouter=require("./routes/cart-router.js");
```

Comment:
Import all the modules at the top and then use . Don't import modules between the code . The code looks pretty then . Just a suggestion